source/adapters/level_zero/device.hpp

```diff
@@ -201,4 +201,7 @@ struct ur_device_handle_t_ : _ur_object {
   ZeCache<struct ze_global_memsize> ZeGlobalMemSize;
   ZeCache<ZeStruct<ze_mutable_command_list_exp_properties_t>>
       ZeDeviceMutableCmdListsProperties;
+
+  // Global (per-adapter) Id of the device
+  size_t Id;
```

Comment:
this is unrelated.

Reply:
I've put this in a separate commit. Do you want me to move it to a separate PR?

Reply:
I don't think we should have this at all yet, since it's dead code.